test/cases/api/container.registry.sh

```diff
@@ -47,8 +48,14 @@ function createReqFile() {
         "baseurl": "$PAYLOAD_REPO_URL"
       }
     ],
+    "containers":[
+      {
+        "source": "$CONTAINER_SOURCE"
+      }
+    ],
     "packages": [
       "postgresql",
+      "podman",
```

Comment:
I wonder whether we should not consider adding some container runtime to the package set automatically if the `containers` customization is specified 🤔

Reply:
Yeah, that is a good idea; I guess we should introduce a customisation for it like we do with the kernel, where one can select the desired container runtime `["podman", "cri-o", ... "docker"].